src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    public String getRandomBall() {
```

Comment:
When the method has name `getRandomBall` and we have class `Ball` it can be confusing if we return `String`, <br>it would be better if return type would be `Ball` (like in your `getRandomColor` method)